page-optimize.php

```diff
@@ -36,10 +36,16 @@ function page_optimize_cache_cleanup( $file_age = DAY_IN_SECONDS ) {
 		return;
 	}
 
+	// If cache is disabled when the cleanup runs, purge it
+	$using_cache = defined( 'PAGE_OPTIMIZE_CACHE_DIR' ) && ! empty( PAGE_OPTIMIZE_CACHE_DIR );
```

Comment:
How will we know what directory to clean up when `PAGE_OPTIMIZE_CACHE_DIR` is indeed empty?

We probably need to update this function to take a cache dir argument and include the current cache dir as an arg when we schedule the event with `wp_schedule_event`.

Thankfully, we can check whether cleanup is already scheduled for a specific cache directory because  `wp_next_scheduled` can look up an event with specific args.

Reply:
It's silly, the moment you posted it, I was "AAAAAAAAAAAH!". :man_facepalming:

Reply:
😄 I can relate